lib/address.js

```diff
@@ -112,6 +112,8 @@ Address.prototype._classifyArguments = function(data, network, type) {
     return Address._transformScript(data, network);
   } else if (typeof(data) === 'string') {
     return Address._transformString(data, network, type);
+  } else if (_.isObject(data) && data.network && data.hash && data.type) {
```

Comment:
I was thinking it would be better to take the path, and then receive precondition error, than to spend time trying to figure out why why the object wasn't being accepted an as input because the property was slightly incorrect.